Sources/MapboxNavigation/NavigationViewControllerDelegate.swift

```diff
@@ -165,14 +165,6 @@ public protocol NavigationViewControllerDelegate: VisualInstructionDelegate {
      */
     func navigationViewController(_ navigationViewController: NavigationViewController, shapeFor waypoints: [Waypoint], legIndex: Int) -> FeatureCollection?
     
-    /**
-     Called when the user taps to select a route on the navigation view controller’s map view.
-     
-     - parameter navigationViewController: The navigation view controller presenting the route that the user selected.
-     - parameter route: The route on the map that the user selected.
-     */
-    func navigationViewController(_ navigationViewController: NavigationViewController, didSelect route: Route)
```

Comment:
Since `NavigationViewController` only allows to show one route this method is not relevant and is not used anywhere.